File: app/src/main/java/croom/konekom/in/a100pitask/rest/ApiInterface.java
package croom.konekom.in.a100pitask.rest;

import java.util.List;

import croom.konekom.in.a100pitask.model.SuccessResponse;
import retrofit2.Call;
import retrofit2.http.GET;

public interface ApiInterface {
    @GET("getcurrencies")
    Call<SuccessResponse> getAllCurrencies();
}

File: app/src/main/java/croom/konekom/in/a100pitask/rest/ApiInterface.kt
package croom.konekom.`in`.a100pitask.rest

import croom.konekom.`in`.a100pitask.model.SuccessResponse
import retrofit2.Call
import retrofit2.http.GET

/***
 * Created By Kartikey Kumar Srivastava
 */

//API to be called
interface ApiInterface {
    @get:GET("getcurrencies")
    val allCurrencies: Call<SuccessResponse>


}
